api/openapi.yaml.in

```diff
@@ -976,6 +976,13 @@ components:
         Options for the Boot Orchestration Service.
       type: object
       properties:
+        cfs_read_timeout:
+          type: integer
+          description: |
+            The amount of time (in seconds) to wait for a response before timing out a request to CFS
+          example: "20"
```

Comment:
I don't know if it matters, but we don't want this to look string-like. :) 

```suggestion
          example: 20
```

Reply:
True. Fixed.